src/char/barbarian.py

```diff
@@ -137,7 +164,16 @@ def kill_council(self) -> bool:
             # Stay inside and cast war cry again moving forward
             self._move_and_attack((40, 10), atk_len)
             self._move_and_attack((-40, -20), atk_len)
-        self._do_hork(5)
+        wait(0.5, 0.55)
```

Comment:
can we further encapsulate this in a form of:
```python
self._do_hork(5, offhand=True)
```
Can hide shenanigans with waiting and swapping and swapping back there